client/app/layout.tsx

```diff
@@ -10,9 +10,13 @@ import ThemeRegistry from "@/components/theme/ThemeRegistry";
 import Header from "@/components/layout/Header";
 import Footer from "@/components/layout/Footer";
 
+// 🏷 import {named} can be significantly slower than import default
+import Box from "@mui/material/Box";
+
 export const metadata = {
   title: "CAS OBPS REGISTRATION",
-  description: "",
+  description:
```

Comment:
Did we get this text from the designers? This might be one of those things gov or the business area has rules about

Reply:
- For testing, I had cut/paste the test descriptions from [ticket 112](https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/112), which only had e2e. I have added unit tests to [ticket 194](https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/194)
- For Tailwind use, implementation is through className, e.g. (I'll add some documentation in the developer guidelines): 
``` 
<TableCell
                      **className**="text-blue-600"
                      align="center"
                      sx={{ border: "1px solid black" }}
                    >
                      {row.event}
                    </TableCell>
```
- For app meta description I have added the detail to [ticket 163](https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/163)

Reply:
1. At certain sizes, the table or images are cut off, or the header/footer doesn't extend all the way across the page. I don't think that's very important for MVP, especially since the business area hasn't decided what content should go on the landing page, so if it's not a quick fix, I'd say don't worry about it.
![Screenshot from 2023-10-17 08-11-05](https://github.com/bcgov/cas-registration/assets/77306764/2f1facfb-8f66-4cb8-bd2b-bf5a1ddfcffe)
![Screenshot from 2023-10-17 08-11-14](https://github.com/bcgov/cas-registration/assets/77306764/4475ef82-f261-4808-9c28-ba449bef6205)
![Screenshot from 2023-10-17 08-11-29](https://github.com/bcgov/cas-registration/assets/77306764/6d2b3f16-9cf1-4931-af6f-cce0bdffa77d)

2. I didn't see a change to the docs, am I missing something?
3. In your testing note `"Prod" url displays components as above...`, what's the prod url?

Reply:
1. Agree on the need to further refine the content page as noted here  [ticket 163](https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/163)
2. Doc update: https://github.com/bcgov/cas-registration/blob/feat/landing-page/112/docs/developer-guide.md
3. Was hoping you knew ;) need to ask Dylan

Reply:
The URL https://cas-reg-frontend-dev.apps.silver.devops.gov.bc.ca/ shows what's in the develop branch (with a delay), so I don't think I'll be able to see your work until after it's merged

Reply:
Thank-you.  I had updated test steps to reflect that, see: **Post Branch Merge**